src/algebra/module/submodule_lattice.lean

```diff
@@ -247,3 +247,39 @@ add_submonoid.to_nat_submodule.apply_symm_apply S
 end nat_submodule
 
 end add_comm_monoid
+
+
+section int_module
+
+variables [add_comm_group M]
+
+/-- An additive subgroup is equivalent to a ℤ-submodule. -/
+def add_subgroup.to_int_submodule : add_subgroup M ≃o submodule ℤ M := add_comm_group.int_module M
```

Comment:
```suggestion
def add_subgroup.to_int_submodule : add_subgroup M ≃o submodule ℤ M :=
```
What happened here?

Reply:
I deleted the wrong thing, then copy pasted the wrong line (